Johnson/johnson.py

```diff
@@ -0,0 +1,87 @@
+# This code is contributed by Riyazul555
+
+# Time Complexity  = O(V * E + V * (V + E) * log(V))  where where V is the number of vertices and E is the number of edges
+# Space Complexity = O(V + E)                         where where V is the number of vertices and E is the number of edges
+
+from queue import Queue
+from heapq import heappush, heappop
+
+nMax = 1005
```

Comment:
same discussion, I think an `nMax` const is not needed in Python, since we can dynamically allocate the structures.